src/ceph-disk

```diff
@@ -1023,7 +1023,10 @@ def update_partition(action, dev, description):
      # On RHEL and CentOS distros, calling partprobe forces a reboot of the
      # server. Since we are not resizing partitons so we rely on calling
      # partx
-     if platform_distro().startswith(('centos', 'red', 'scientific')):
+
+     # This was fixed in rhel 7.x, checking for it.
+     rel = platform_information()[1]
+     if float(rel) < 7.0 and platform_distro().startswith(('centos', 'red', 'scientific')):
```

Comment:
this doesn't need a call to `float()` here as it can potentially break (as @dachary points in the comments).

A simple `startswith('7') would be enough.

Also, calling the version `rel` is misleading. I would suggest to keep the first `if` condition and then add the version
comparison afterwards to make it explicit that this is only intended for RHEL:

    # On RHEL and CentOS distros, calling partprobe forces a reboot of the
    # server. Since we are not resizing partitons so we rely on calling
    # partx
    if platform_distro().startswith(('centos', 'red', 'scientific')):
        # This was fixed in rhel 7.x, checking for it.
        _, version, _ = platform_information()
        if not version.startswith('7'):